forte/ontology_specs/wikipedia_ontology.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "wikipedia",
   "imports": [
-    "ft.onto.base_ontology"
+    "base_ontology.json"
```

Comment:
Is this .json?

Reply:
Further, how can one distinguish a package from a file path? From the string surface, it is impossible. I would rather suggest use two separate keys.

Reply:
Yes, it should be .json, in this case, we would want to use base_ontology.json from the same directory, and not some other base_ontology version that might be installed. 

In the next commit, I have added separate keys for filepath and package.